.github/workflows/release-on-milestone-closed.yml

```diff
@@ -0,0 +1,61 @@
+name: "Automatic Releases"
+
+on:
+  workflow_call:
+    inputs:
+      use-next-minor-as-default-branch:
+        description: "Whether to switch the default branch to next minor"
+        default: false
+        required: false
+        type: "boolean"
+
+jobs:
+  release:
```

Comment:
This one looks generic enough that it could be defined directly in `laminas/automatic-releases`, but it seems impossible because of [this limitation](https://github.com/doctrine/orm/pull/9114#issuecomment-939457664), which does not seem to be documented at https://docs.github.com/en/actions/learn-github-actions/reusing-workflows